test/api/flat-dedupes.test.js

```diff
@@ -1,56 +1,72 @@
-import { Generator } from '@jspm/generator';
-import { strictEqual } from 'assert';
+import { Generator } from "@jspm/generator";
+import { strictEqual } from "assert";
 
 const BASE_CONFIG = {
-  mapUrl: 'about:blank',
+  mapUrl: "about:blank",
   ignore: [
-    'react',
-    'react/jsx-runtime',
-    'react-dom',
-    'react-dom/server',
-    'framer',
-    'framer-motion',
-    'framer-motion/three',
+    "react",
+    "react/jsx-runtime",
+    "react-dom",
+    "react-dom/server",
+    "framer",
```

Comment:
I ran `prettier` on this file, hence all the odd changes - again, happy to revert. Also happy to go wild and run prettier on everything 👹

Reply:
Happy to run prettier globally!